cmd/chase.go

```diff
@@ -240,7 +245,7 @@ func parseStdInList(list []string, operating *string) (packages.IPackage, error)
 	default:
 		logLady.WithFields(logrus.Fields{
 			"list": list,
-		}).Trace("Chasing Yum")
+		}).Trace("Chasing yum")
```

Comment:
Maybe for clarity while we are at it would you mind adding something like `(default case)` to the `Chasing yum` so that if looking at the debug log we know that we fell all the way through the switch